src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -30,6 +30,35 @@
 #include "utilities/macros.hpp"
 #include "utilities/sizes.hpp"
 
+#define MAX_FEATURE_VEC_SIZE 4
+
+class VM_Features {
+ public:
+  using FeatureVector = uint64_t [MAX_FEATURE_VEC_SIZE];
```

Comment:
Do you think it would be better to refactor this into a separate class analogous to `std::bitset`? You can start with only implementing `test`, `set`, `reset`. This would help in other use cases, too.

https://en.cppreference.com/w/cpp/utility/bitset

Reply:
In essence, what we have currently is a bitmap implementation, but its utility is limited to VM_Version for now. The current approach simplifies the JVMCI side of handling. We have an existing utility for bitset src/hotspot/share/utilities/bitMap.hpp